weave/flow/obj.py

```diff
@@ -23,6 +45,7 @@ class Object(BaseModel):
         arbitrary_types_allowed=True,
         protected_namespaces=(),
         extra="forbid",
+        populate_by_name=True,
```

Comment:
not necessary, but I had to look up what this meant, might be nice to add a comment about what this does and why we turned it on.

Reply:
(this will affect all aliases right?)

Reply:
Yeah, affects all aliases, though I don't think we use any atm.  In future it will be used to support deprecating fields.